src/workspace/mod.rs

```diff
@@ -141,6 +141,8 @@ pub enum WorkspaceDiagnosticKind {
     previous: bool,
     suggestion: NodeModulesDirMode,
   },
+  #[error("invalid workspace member \"{name}\".")]
+  InvalidMember { name: String },
```

Comment:
```suggestion
  #[error("Invalid workspace member name \"{name}\". Ensure the name is in the format '@scope/name'.")]
  InvalidMemberName { name: String },
```

Let's give an example to help out a bit and it looks like one other diagnostic uses a capital letter here.